apps/cookbook/src/app/showcase/badge-showcase/badge-showcase.component.html

```diff
@@ -1,15 +1,34 @@
 <p>
-  Badges are inlines block elements that often are used on top of or close to another component.
-  They can be used with numbers, text and icons. A small badge version also exists which does not
-  support slotted content but can be used to indicate status.
+  Badges are inline block elements that often are used on top of or close to another component. They
+  can be used with numbers, text and icons. A small badge version also exists which does not support
+  slotted content.
+</p>
+
+<kirby-divider [hasMargin]="true"></kirby-divider>
+
+<h2>Elevation on white badges</h2>
+<p>
+  A white badge can easily blend into the background and just appear as a number, icon or not at all
+  if no content is passed. Here it can be necesssary to apply some elevation to the badge; this is
+  also done for the examples below where all the badges are displayed on a white background.
+</p>
+<p>
+  The <code>--kirby-badge-elevation</code> property can be useful for this. You can see how it is
+  applied to the examples below by inspecting
+  <a
+    href="https://github.com/kirbydesign/designsystem/blob/master/apps/cookbook/src/app/examples/badge-example/badge-example.component.scss"
+    target="_blank"
+  >
```

Comment:
This link should be to the badge-example-shared.scss instead, shouldn't it? 

Also consider adding kirby-external-icon to this bad-boy for that extra pizzazz!